src/http2/aws-http2-stream.ads

```diff
@@ -87,7 +90,8 @@ package AWS.HTTP2.Stream is
      (Self  : in out Object;
       Ctx   : in out Server.Context.Object;
       Frame : HTTP2.Frame.Object'Class;
-      Error : out Error_Codes)
+      Error : out Error_Codes;
+      Info  : out Natural)
```

Comment:
Looks like Info was some kind of debug data and used only as information into a go-away frame. Can we put this data into the Stream itself and have an function to access this value? This clobber a bit the API for nothing semantic.

Even better would be to have an enumerated type with clear state information. Do you think it can be done?

Reply:
Yes, I did.